src/main/resources/static/swagger.yaml

```diff
@@ -257,7 +257,7 @@ components:
                 dataExpiryAt:
                   type: string
                   format: date-time
-    ConsentStatus:
+    NotificationType:
```

Comment:
Not sure about name "NotificationType". It really is not a "type" as such. I would have agreed to it, if it was an in the contract "type" is an object, with different attributes or meaning. Its just set of values like REQUSTED, EXPIRED, DENIED, REVOKED etc. Basically status of either consent-request or consent-artefact. So, I think "ConsentStatus" is a right name. 
"ConsentNotificationStatus" but that too, IMO, is redundant. 

This probably also should make us think in the contract should we be calling it "type". maybe the property name really should be "status"